xml/Microsoft.JScript.Vsa/ResInfo.xml

```diff
@@ -82,10 +82,10 @@
         <Parameter Name="isLinked" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="filename">To be added.</param>
-        <param name="name">To be added.</param>
-        <param name="isPublic">To be added.</param>
-        <param name="isLinked">To be added.</param>
+        <param name="filename">Specifies a filename.</param>
+        <param name="name">Specifies a name.</param>
+        <param name="isPublic">Indicates whether the member is public.</param>
+        <param name="isLinked">Indicates the item is linked.</param>
```

Comment:
```suggestion
        <param name="isLinked">
          <see langword="true" /> to indicate the resource is linked; otherwise, <see langword="false" />.</param>
```